components/server/src/prebuilds/bitbucket-app.ts

```diff
@@ -130,6 +130,20 @@ export class BitbucketApp {
                 try {
                     const projectOwner = await this.findProjectOwner(project, user);
 
+                    if (project.settings?.prebuilds?.triggerStrategy === "activity-based") {
+                        await this.projectService.updateProject(projectOwner, {
```

Comment:
Hm, why doing these reverts? :thinking: 
I can only imagine them doing more harm than good...

Reply:
I'd like the configured trigger strategy to actually reflect the status of external webhooks. If they are still configured, even after toggling prebuilds on and off, it would be nice to not misreport that as them being activity-based.

Happy to chat about it